### DISCUSSION TITLE
Get token from request is truncated

### DISCUSSION THREAD

**Discussion by colcherica** [cypress-io/cypress#24940] [Questions and Help]
When I am trying to get the token from my request I saw in cy.log that the token is truncated: 
This is how I save my token: 
![image](https://user-images.githubusercontent.com/20081012/205285110-673e3c76-45a2-40a3-b5ae-d2e500e15e55.png)

I tried with token generated via postman and is working , but with one done via cypress.request is not working

**Comment by jjhelguero** [comment]
Did you console.log `response.body.auth_token` to make sure it isn't truncated?

**Reply by colcherica** [reply]
this is my first request to have the token: 
![image](https://user-images.githubusercontent.com/20081012/205327988-04767db3-54bb-480f-8d7e-e41209029ed5.png)

and this is the token from log: 
![image](https://user-images.githubusercontent.com/20081012/205328370-db4adee5-4943-4c4a-86f4-136c94ed93ae.png)

if I will add this token into my postman request I will receive an error

**Reply by colcherica** [reply]
> Did you console.log `response.body.auth_token` to make sure it isn't truncated?

yes, I took the generated token from response and added into postman request and it not working

**Reply by colcherica** [reply]
> > Did you console.log `response.body.auth_token` to make sure it isn't truncated?
> 
> yes, I took the generated token from response and added into postman request and it not working
The request has https is that something that I need to put into headers in order to pass the https request ?
